### PR TITLE
[ioctl/newcmd] modify GetAddress to DefaultAddress 

### PR DESCRIPTION
**Pull request by huof6829**

### CODE REVIEW

File: ioctl/client.go [hunk 1]
```diff
@@ -46,9 +50,9 @@ type (
 		ReadSecret() (string, error)
 		// Execute a bash command
 		Execute(string) error
-		// doing
-		GetAddress(in string) (string, error)
-		// doing
+		// DefaultAddress returns default address if input empty
+		DefaultAddress(in string) (string, error)
```
Comment (CoderZhi) [inline ioctl/client.go:54]: the name is still not clear. how about `AddressWithDefaultIfNotExist`

File: ioctl/client.go [hunk 2]
```diff
@@ -147,8 +151,34 @@ func (c *client) SelectTranslation(trls map[config.Language]string) (string, con
 }
 
 func (c *client) ReadSecret() (string, error) {
-	// TODO: delete util.ReadSecretFromStdin, and move code to here
-	return util.ReadSecretFromStdin()
+	signalListener := make(chan os.Signal, 1)
```
Comment (CoderZhi) [inline ioctl/client.go:154]: revert this change. you can do it in a different PR or even leave as it is until we migrate to the new command

File: ioctl/client.go (hunk 1 above)
Comment (huof6829) [reply ioctl/client.go:54]: ok

File: ioctl/client.go (hunk 2 above)
Comment (huof6829) [reply ioctl/client.go:154]: ok

**Review by CoderZhi — APPROVED** [review]

**Review by Liuhaai — APPROVED** [review]
